site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -1,23 +1,11 @@
 ---
-title: "Deploy GKE/AKS cluster and install Agones using Terraform"
-linkTitle: "Install with Terraform"
-weight: 4
+title: "Installing Agones on Google Kubernetes Engine"
+linkTitle: "Google Cloud"
+weight: 10
 description: >
-  Install a [Kubernetes](http://kubernetes.io) cluster and Agones declaratively using Terraform.
-
+  You can use Terraform to provision a GKE cluster and install Agones on it.
 ---
 
-## Prerequisites
-
-- Terraform v0.12.3
-- [Helm](https://docs.helm.sh/helm/) package manager 2.10.0+
-- Access to the the Kubernetes hosting provider you are using (e.g. `gcloud` or `az` utility installed)
-- Git
-
-## Installing Agones on Google Kubernetes Engine using a Terraform submodule
-
-You can use Terraform to provision a GKE cluster and install Agones on it.
-
 The first step is to enable the `Kubernetes Engine API`. From the Cloud Console, navigate to APIs & Services > Dashboard, then click `Enable APIs and Services`. Type `kubernetes` in the search box to find the Kubernetes Engine API. Click Enable.
```

Comment:
I'd suggest coping the first part of the before you begin section from https://7293852-dot-preview-dot-agones-images.appspot.com/site/docs/installation/creating-cluster/gke/ so that we have consistent steps to get started on GCP.

Reply:
Nice. Doing!